### PR TITLE
Add composer file

### PR DESCRIPTION
**Pull request by zahara**
Along with the version tag, this should fix issue #13 - am I missing anything @mwjames?

### CODE REVIEW

**Comment by zahara** [comment]
Also need to set the homepage and choose a license

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	"description": "RDFIO extension for importing data in RDF format to Semantic MediaWiki",
+	"keywords": [
+		"MediaWiki",
+		"Semantic MediaWiki",
+		"RDF",
+		"Linked Data"
+	],
+	"homepage": "<attempt to pull from ExtensionName.php, else blank>",
+	"license": "<leave as ‘unknown’?>",
+
```
Comment (mwjames) [inline composer.json]: Empty line.

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+	"name": "mediawiki/rdfio",
```
Comment (mwjames) [inline composer.json]: Once merged, please make sure to register with [0].

[0] https://packagist.org/

**Comment by mwjames** [comment]
All objects referencing to a `global` in `RDFIO.php` should be changed to use the explicit form `GLOBALS` such as `$wgExtensionCredits` to `GLOBALS['wgExtensionCredits']`.

```
$smwgARC2Path = $smwgIP . '/libs/arc/';
$smwgARC2MainFile = $smwgARC2Path . '/ARC2.php';
if ( ! file_exists( $smwgARC2MainFile )) {
    throw new MWException("ARC2 library is not installed in Semantic Mediawiki libs folder! Please refer to the installation instructions, to fix that: http://www.mediawiki.org/wiki/Extension:RDFIO#Installation");
} else {
    require_once( $smwgARC2MainFile );
}
```

Could be removed since arc2 is installed via Composer and uses the same autoloader.

Hard coded references like `/libs/arc/` should be removed as with Composer, arc2 is installed as library in the `vendor` directory and autoloaded.

**Comment by samuell** [comment]
I leave it all up to @zahara now, as the new maintainer, to check this! :)

**Comment by samuell** [comment]
Is this just waiting for me to merge, @zahara ? 

Sorry, I was not sure what workflow we were wanting to go with. I thought you might as well do the merges, as a maintainer? :)

**Comment by samuell** [comment]
@zahara I merged this this time, as I need it for my [RDFIO vagrant box](https://github.com/samuell/devbox-rdfio) I'm working on ... :P

But do let me know what workflow you'd prefer for pull requests! I'm not sure whether we did decide clearly the workflow to go about with that?

(Btw, it seem I will have time to do a little work on it in the nearest, as part of some research efforts in our group ...)

// Cheers

**Comment by samuell** [comment]
The `composer.json` seems to have some minor schema errors:

```bash
[vagrant RDFIO]$ composer run-script composer.json 


                                                              
  [Composer\Json\JsonValidationException]                     
  "./composer.json" does not match the expected JSON schema:  
   - homepage : Invalid URL format                            
```

**Pull request merged**